tests/framework/jailer.py

```diff
@@ -63,11 +65,19 @@ def __init__(
         self.daemonize = daemonize
         self.extra_args = extra_args
         self.api_socket_name = DEFAULT_USOCKET_NAME
+        self.cgroups = cgroups
+
+        if self.cgroups is None:
+            self.cgroups = ['cpuset.mems=0', 'cpuset.cpus=0']
```

Comment:
Here I was just trying to emulate the `--node` behavior, as `--node` have been added again I guess we can delete this but maybe is something to take into account once `--node` is totally removed.

Reply:
> Here I was just trying to emulate the `--node` behavior, as `--node` have been added again I guess we can delete this but maybe is something to take into account once `--node` is totally removed.

Wouldn't this impact the integration tests, since they are going to run with a Firecracker microVM which is confined to a cpuset with only one CPU core?

Reply:
You're right, I didn't think on that. I'll let the default value to `None` as you said.